### PR TITLE
Add option to use my favorite authors to do a project

### PR DESCRIPTION
**Pull request by LaetitiaRiffaud**

### CODE REVIEW

File: Tests/Fixtures/Project/src/Worldia/Bundle/ProductTestBundle/Service/Project/Author.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        ];
+    }
+
+    public function setPage($page)
+    {
+        $this->page = (null === $page ? $page : (int)$page);
+
+        return $this;
+    }
+}
```
Comment (Pitoune) [inline Tests/Fixtures/Project/src/Worldia/Bundle/ProductTestBundle/Service/Project/Author.php]: new line at EOF

File: Tests/Units/Translation/TranslationManagerTest.php [hunk 2]
```diff
@@ -56,6 +56,10 @@ public function shouldCreateProject()
         $projectMock->expects($this->once())
             ->method('save')
             ->willReturn($projectMock);
+        $projectMock->expects($this->once())
+            ->method('getPotentialAuthors')
+            ->with('my_textmaster')
+            ->willReturn(['55c3763e656462000b000027']);
```
Comment (Pitoune) [inline Tests/Units/Translation/TranslationManagerTest.php]: According to the [php doc](https://github.com/worldia/textmaster-api/blob/64c5340a839f1bd49f3c463458d51b8198bb4ea6/lib/Textmaster/Model/ProjectInterface.php#L283) the method `getPotentialAuthors` will not return an array of string.

File: Translation/TranslationManager.php [hunk 3]
```diff
@@ -82,10 +82,14 @@ public function create(
             ->setOptions($options)
             ->setWorkTemplate($workTemplate)
             ->setCallback($this->generateProjectCallback())
-            ->setTextmasters($textmasters)
         ;
 
         $project->save();
+
+        if ($useMyTextmasters) {
+            $project->setTextmasters($project->getPotentialAuthors('my_textmaster'));
```
Comment (Pitoune) [inline Translation/TranslationManager.php]: Not sure it is gonna work as `getPotentialAuthors` returns an array of `AuthorInterface`

File: Tests/Fixtures/Project/src/Worldia/Bundle/ProductTestBundle/Service/Project/Author.php (hunk 1 above)
Comment (LaetitiaRiffaud) [inline Tests/Fixtures/Project/src/Worldia/Bundle/ProductTestBundle/Service/Project/Author.php]: Done

File: Tests/Units/Translation/TranslationManagerTest.php (hunk 2 above)
Comment (LaetitiaRiffaud) [inline Tests/Units/Translation/TranslationManagerTest.php]: You're right.

File: Translation/TranslationManager.php (hunk 3 above)
Comment (LaetitiaRiffaud) [inline Translation/TranslationManager.php]: Yes, of course.

**Pull request merged**